geode-core/src/distributedTest/java/org/apache/geode/internal/cache/DeltaForceSizingFlagDUnitTest.java

```diff
@@ -0,0 +1,995 @@
+    });
+
+  }
+
+  private void createRRHeapLRU(MemberVM memberVM) {
+    memberVM.invoke("Create rr-heap-lru", () -> {
+      Cache cache = ClusterStartupRule.getCache();
+      ResourceManager manager = cache.getResourceManager();
+      manager.setCriticalHeapPercentage(95);
+      manager.setEvictionHeapPercentage(90);
+      AttributesFactory<Integer, TestDelta> attr = new AttributesFactory<Integer, TestDelta>();
+      attr.setEvictionAttributes(EvictionAttributes.createLRUHeapAttributes(new TestObjectSizer(),
+          EvictionAction.OVERFLOW_TO_DISK));
+      attr.setDiskDirs(getMyDiskDirs());
+      attr.setDataPolicy(DataPolicy.REPLICATE);
+      attr.setScope(Scope.DISTRIBUTED_ACK);
+      attr.setDiskDirs(getMyDiskDirs());
+      Region region = cache.createRegion("region", attr.create());
+    });
+
+  }
+
+  private void createPRHeapLRU(MemberVM memberVM) {
+    memberVM.invoke("Create pr-heap-lru", () -> {
+      Cache cache = ClusterStartupRule.getCache();
+      ResourceManager manager = cache.getResourceManager();
+      manager.setCriticalHeapPercentage(95);
+      manager.setEvictionHeapPercentage(90);
+
+      AttributesFactory<Integer, TestDelta> attr = new AttributesFactory<Integer, TestDelta>();
+      PartitionAttributesFactory<Integer, TestDelta> paf =
+          new PartitionAttributesFactory<Integer, TestDelta>();
+      paf.setRedundantCopies(1);
+      attr.setEvictionAttributes(EvictionAttributes.createLRUHeapAttributes(new TestObjectSizer(),
+          EvictionAction.LOCAL_DESTROY));
+      PartitionAttributes<Integer, TestDelta> prAttr = paf.create();
+      attr.setPartitionAttributes(prAttr);
+      attr.setDataPolicy(DataPolicy.PARTITION);
+      attr.setSubscriptionAttributes(new SubscriptionAttributes(InterestPolicy.ALL));
+      Region<Integer, TestDelta> region = cache.createRegion("region", attr.create());
+    });
+
+  }
+
+  private static class TestObjectSizer implements ObjectSizer {
+    private AtomicInteger invocations = new AtomicInteger();
+
+    @Override
+    public int sizeof(Object o) {
+      if (InternalDistributedSystem.getLogger().fineEnabled()) {
+        InternalDistributedSystem.getLogger()
+            .fine("TestObjectSizer invoked"/* , new Exception("stack trace") */);
+      }
+      if (o instanceof TestObject) {
+        invocations.incrementAndGet();
+        return ((TestObject) o).sizeForSizer;
+      }
+      if (o instanceof TestDelta) {
+        invocations.incrementAndGet();
+        return ((TestDelta) o).info.length();
+      }
+      if (o instanceof Integer) {
+        return 0;
+      }
+      if (o instanceof TestKey) {
+        invocations.incrementAndGet();
+        return ((TestKey) o).value.length();
+      }
+      throw new RuntimeException("Unpected type to be sized " + o.getClass() + ", object=" + o);
+    }
+  }
+
+  private static class TestKey implements DataSerializable {
+    String value;
+
+    public TestKey() {
```

Comment:
The compiler warning here can be fixed by adding a `@SuppressWarnings("unused")` annotation above the constructor.